quark/script/ciphey.py

```diff
@@ -2,14 +2,23 @@
 # This file is part of Quark-Engine - https://github.com/quark-engine/quark-engine
 # See the file 'LICENSE' for copying permission.
 
-from ciphey import decrypt
-from ciphey.iface import Config
-
 
 def checkClearText(inputString: str) -> str:
-    """Check the decrypted value of the input string.
+    """Check the decrypted value of the input string with Ciphey.
+    If Ciphey is not installed, raise ImportError.
 
     :param inputString: string to be checked.
```

Comment:
Please refer to [the Sphinx docstring format](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format) and add the exception description between lines 10 and 11.